### ISSUE TITLE
Not able to scroll a long modal

### ISSUE THREAD

**Issue by midnightcodr** [smooth-code/smooth-ui#128]
## 🐛 Bug Report

https://wkx04p7yp8.codesandbox.io/
## To Reproduce

Please go to https://wkx04p7yp8.codesandbox.io/, switch to mobile mode, then try to scroll the content using a pointer device (it will be a user's finger for a real mobile device), you will find that scrolling is impossible using a pointer device with two exceptions:
1) you can only scroll using a browser's vertical scroll bar
2) you can scroll using a keyboard's up and down keys

Neither of which is possible using a mobile device. I've tested this URL using a real phone.

## Expected behavior
Scrolling should work with a pointer device.

## Link to repl or repo (highly encouraged)
https://wkx04p7yp8.codesandbox.io/

**Comment by midnightcodr** [comment]
Never mind, I was not using the data type for attribute `overflow`. Fixed.

**Comment by neoziro** [comment]
Actually it is a true bug, visible in documentation. I think it is relative to scroll blocking, cc @theKashey.

**Comment by theKashey** [comment]
Yeah. It’s partially my fault. Lock should allow _some_ move if target is greater than parent. Just a few lines no fix in conditions.

**Comment by theKashey** [comment]
Could be easily fixed with a proper overflow, but better to be solved on a lover level

**Comment by theKashey** [comment]
There was a mistake in scroll _possibility_ calculations, which led to a false negative in case of `smooth-ui`. 
Please update `react-remove-scroll` - just remove it from `yarn.lock` or `package.lock` and reinstall packages - https://codesandbox.io/s/6jqlqwmllk

**Comment by neoziro** [comment]
Modal is gone in new release, please use [Reakit Dialog](https://reakit.io/docs/dialog/) instead.
